### PR TITLE
Add code coverage

### PR DESCRIPTION
**Pull request by moedu23**
Nice work so far folks! Code coverage is at 98.76%

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [![Build status](https://travis-ci.org/epicodus/epicenter.svg?branch=master)](https://travis-ci.org/epicodus/epicenter)
 [![Code climate](https://codeclimate.com/github/epicodus/epicenter/badges/gpa.svg)](https://codeclimate.com/github/epicodus/epicenter)
+[![Coverage Status](https://coveralls.io/repos/epicodus/epicenter/badge.svg)](https://coveralls.io/r/epicodus/epicenter)
```
Comment (michaelrkn) [inline README.md]: lowercase "s" in "status" please

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,3 +1,12 @@
+require 'simplecov'
+require 'coveralls'
```
Comment (michaelrkn) [inline spec/spec_helper.rb]: what's the point of "require: false" in the Gemfile when you just require it here?

File: Gemfile [hunk 3]
```diff
@@ -45,6 +45,8 @@ group :test do
   gem 'webmock', require: false
   gem 'vcr'
   gem 'puffing-billy'
+  gem 'simplecov', require: false
+  gem 'coveralls', require: false
```
Comment (michaelrkn) [inline Gemfile]: why do we need simplecov and coveralls?

Comment (moedu23) [inline Gemfile]: In order for SimpleCov and Coveralls to run, the `Simplecov.start` call has to occur before any of the application code is required, so should be at the very top of `spec_helper.rb`

Comment (moedu23) [inline Gemfile]: And by run, I should say that they're starting to track files and evaluate their coverage.

**Comment by moedu23** [comment]
Simplecov is tracking files and evaluating their test coverage. Coveralls will help us track code coverage over time, to ensure that we're keeping test coverage at or near 100%. Using the two together will help us to target code that's not covered by tests, and ensure that we're not merging in untested code. For an open source project, with many contributors, it's an ongoing reminder to keep test coverage standards up.

I'll update the README and investigate the TravisCI build failure this evening!

**Comment by moedu23** [comment]
Looking into the Travis-ci failure, it looks like `credit_card_pages_spec.rb:22`could be a flaky test. Since it's not related to this PR I'll push my branch again to see if it passes on Travis this time. And create a new issue to investigate this test.

**Pull request merged**